packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -402,71 +409,114 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
 
     return (
       <>
-        <div className="flex justify-center">
-          <div className="min-w-80 bg-default border-subtle w-full rounded-md border p-7">
-            <div className="flex">
-              <div className="bg-subtle text-default mt-[3px] flex h-5 w-5 items-center justify-center rounded-full p-1 text-xs font-medium ltr:mr-5 rtl:ml-5">
-                1
-              </div>
-              <div>
-                <div className="text-emphasis text-base font-bold">{t("trigger")}</div>
-                <div className="text-default text-sm">{t("when_something_happens")}</div>
-              </div>
-            </div>
-            <div className="border-subtle my-7 border-t" />
-            <Label>{t("when")}</Label>
-            <Controller
-              name="trigger"
-              control={form.control}
-              render={() => {
-                return (
-                  <Select
-                    isSearchable={false}
-                    className="text-sm"
-                    id="trigger-select"
-                    isDisabled={props.readOnly}
-                    onChange={(val) => {
-                      if (val) {
-                        form.setValue("trigger", val.value);
-                        const newTimeSectionText = getTimeSectionText(val.value, t);
-                        if (newTimeSectionText) {
-                          setTimeSectionText(newTimeSectionText);
-                          if (
-                            val.value === WorkflowTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW ||
-                            val.value === WorkflowTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW
-                          ) {
-                            form.setValue("time", 5);
-                            form.setValue("timeUnit", TimeUnit.MINUTE);
-                          } else {
-                            form.setValue("time", 24);
-                            form.setValue("timeUnit", TimeUnit.HOUR);
-                          }
+        <div className="mb-3">
```

Comment:
![Image](https://github.com/user-attachments/assets/408eb5db-4fe2-4a24-86e3-c9b5561468d9)

the container should have less padding like this

![Image](https://github.com/user-attachments/assets/a1d3004e-29f4-44a3-9965-5947d7dac1d3)